docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/self-describing-events/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Self-describing events"
+---
+
+### Self-describing events
+
+[Structuring your data with schemas](/docs/understanding-your-pipeline/schemas/index.md) to perform self-describing event tracking is the defacto way to track events with Snowplow and allows any arbitrary name: value pairs to be captured with the event.
+
+:::info
+
+Historically, custom self-describing events were called “unstructured” and the legacy custom events were called “structured”. This terminology can be confusing, so we don’t use it anymore. However, you might find its remnants in some of the APIs.
+
+:::
+
+An example of a self-describing event for a product view event:
+
+```json
+{
+  "schema": "iglu:com.my_company/viewed_product/jsonschema/1-0-0",
```

Comment:
Wouldn't this need to contain `self-desc` there? I have no idea how it works but I had trouble understanding the diff based on this example alone and I saw that in one of the implementations. (cwv i think)